### DISCUSSION TITLE
ITCSS + Jekyll

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#225] [Perguntas e Respostas]
pessoal procurando por uma estrutura já pronta pela web, encontrei [este repositório aqui](https://github.com/neilspurgeon/itcss-jekyll-boilerplate) bem útil para quem quer ter uma noção de como juntar itcss com jekyll... por enquanto o repositório tem so a estrutura de arquivos, o autor não adicionou novas updates, mas com o que já tem, já da para ir usando. então? o que acham?

link: https://github.com/neilspurgeon/itcss-jekyll-boilerplate

**Comment by kandebonfim** [comment]
@leandroruel achei massa!

Vou deixar aqui um [projeto open source](http://itcsscli.com/) meu que ajuda a administrar a estrutura do ITCSS. Uma das paradas que quero fazer é justamente adaptar para o Jekyll já que ele tem uma organização que exige que os módulos ITCSS fiquem numa pasta diferente do arquivo de importação.

**Comment by leandroruel** [comment]
@kandebonfim rapz! show de bola seu projeto :+1:
